docs/osf-trivia-list/deployment.md

```diff
@@ -12,20 +12,46 @@ Github](https://github.com/Dasharo/dasharo-issues).
 
 Following sections explain how to deal with most common `flashrom` problem.
 
-### How to install flashrom ?
+### How to install Dasharo flashrom fork ?
 
-* Install flashrom v1.1 or newer with your distribution's package manager if
-  you don't have it installed yet. If your distro doesn't provide flashrom or
-  provides an outdated one, you can build it yourself using
-  [this instruction](https://www.flashrom.org/Downloads).
-* Or compile recent version of flashrom:
+Following procedure is for advanced users who are familiar with source
+compilation and can resolve potential missing dependencies or other typical
+compilation problems. If you are not familiar with compilation from source we
+recommend use of Dasharo Tools Suite, which is minimal Linux environment to
+deploy, update, and maintain firmware on Dasharo-supported devices. Please
+follow [DTS documentation](../../dasharo-tools-suite/overview). Otherwise
+please follow compilation procedure.
 
-  ``` console
-  sudo apt install libpci-dev libftdi-dev libusb-1.0-0-dev
-  git clone https://github.com/flashrom/flashrom.git
-  cd flashrom
-  sudo make install
-  ```
+Currently, the latest flashrom release lacks support for Alder Lake
```

Comment:
Upstream flashrom has CML-U and ADL-S support but distro may have not picked it up yet. But if we mean a tag when mentioning release , then it is no longer true:

https://github.com/flashrom/flashrom/blob/v1.3.0/chipset_enable.c#L2080
https://github.com/flashrom/flashrom/blob/v1.3.0/chipset_enable.c#L2176

However, the fork may still be needed when doing EC transition

Reply:
@miczyg1 ok, I'm just copying your word  from https://github.com/Dasharo/docs/commit/54b60cc823d1cf5b60e4e8a86b19b8fef190b67c

Please suggest improvements.

Reply:
@miczyg1 please let me know WDYT about: https://github.com/Dasharo/docs/pull/513/commits/eec25a302107ec9d7aa7ed8ddc64d03e054c59c3

Reply:
> @miczyg1 ok, I'm just copying your word from [54b60cc](https://github.com/Dasharo/docs/commit/54b60cc823d1cf5b60e4e8a86b19b8fef190b67c)
> 
> Please suggest improvements.

This commit is from April 2022. The world happened to change countless times since then.

> @miczyg1 please let me know WDYT about: [eec25a3](https://github.com/Dasharo/docs/commit/eec25a302107ec9d7aa7ed8ddc64d03e054c59c3)

I like that

Reply:
@miczyg1, you resolved the discussion, but because of this discussion and request for change whole PR is not approved and is not merged.

Reply:
@pietrushnic I think other discussion were still open back then that is why I didn't approve it yet